src/features/left-sidebar/icons/IconBoxRelay.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import AccessibleSVG from '../../../icons/accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    selected?: boolean,
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const iconName = 'icon-box-relay';
+
+const IconBoxRelay = ({ className = '', width = 14, color = '#c4c4c4', selected = false, title }: Props) => (
```

Comment:
Can this default color come from the variables file?

Reply:
This follows the pattern established by the other leftnav icons (see `IconAllFiles`, `IconFavorites`, `IconNotes`). The purpose of this PR is to solve the addition of a new icon, not to refactor the codebase into constants references.